digital_logic_simulator/settings.py

```diff
@@ -21,17 +21,19 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = ''
```

Comment:
Secret and debug lines shouldn't be edited. Keep their actual values in your .env file.